test/md_zip_view_test.cpp

```diff
@@ -619,3 +620,96 @@ TEST(MDZipView, create_from_labels_nested) {
     base += 1.0;
   }
 }
+
+TEST(MDZipView, event_lists_mutable) {
```

Comment:
Would be good to see some examples with weighted/compressed events being used too.

Reply:
This is "blocked" by all the TODOs given in the PR message. If we do it now we would end up redoing it once the underlying proxy code is refactored.